src/stores/useHolochainStore.js

```diff
@@ -9,28 +9,34 @@ import { hposHolochainCall } from '../services/hpos'
 
 const HC_APP_TIMEOUT = 35_000
 
+const __HC_LAUNCHER_ENV__ = "__HC_LAUNCHER_ENV__";
+const isLauncher = () => globalThis.window && __HC_LAUNCHER_ENV__ in globalThis.window;
+const getLauncherEnvironment = () => isLauncher() ? globalThis.window[__HC_LAUNCHER_ENV__] : undefined;
+
 const makeUseHolochainStore = ({ installed_app_id, app_ws_url, hc_admin_port }) => defineStore('holochain', {
   state: () => ({
     client: null,
+    signingCredentials: null,
     // These two values are subscribed to by clientStore
     appInfo: null,
     isReady: false,
-    signingCredentials: null
   }),
   getters: {
     isAnonymous: _ => false, // for compatibility with holo
-    agentEmail: _ => null, // for compatibility with holo
+    agentEmail: _ => null, // for compatibility with holo,
+    isLauncher,
   },
   actions: {
     // BEGIN useInterfaceStore methods
 
     async initialize() {
       try {
-        const holochainClient = await AppWebsocket.connect(
-          app_ws_url,
-          HC_APP_TIMEOUT,
-          signal => useSignalStore().handleSignal(presentHcSignal(signal))
-        )
+        const holochainClient = await AppWebsocket.connect({
```

Comment:
I don't remember where we landed on this?  Are we still supporting the pure hc case?  

If so, we will need a clause to check if the env is in launcher here and if not, then make the call to fetch the token and send it in with the app connection call too.

Reply:
We decided to not support native HC. I've removed all the relevant code